components/workbenches/workbenches.go

```diff
@@ -40,6 +41,26 @@ type Workbenches struct {
 	components.Component `json:""`
 }
 
+func (w *Workbenches) Init(ctx context.Context, _ cluster.Platform) error {
+	log := logf.FromContext(ctx).WithName(ComponentName)
+
+	var imageParamMap = map[string]string{
+		"odh-notebook-controller-image":    "RELATED_IMAGE_ODH_NOTEBOOK_CONTROLLER_IMAGE",
+		"odh-kf-notebook-controller-image": "RELATED_IMAGE_ODH_KF_NOTEBOOK_CONTROLLER_IMAGE",
+	}
```

Comment:
nitty-gritty :
we could have build two maps here, and do the ApplyParams() call on each.
since there is a range loop, it can just run one value each time.